src/java.desktop/macosx/native/libawt_lwawt/java2d/metal/MTLVertexCache.h

```diff
@@ -32,7 +32,7 @@
 /**
  * Constants that control the size of the vertex cache.
  */
-#define MTLVC_MAX_INDEX         1536
+#define MTLVC_MAX_INDEX         250
```

Comment:
It would be helpful to add a comment about why the max index is 250. Do you think it is worth filing a follow-up issue?

Reply:
Hi Kevin,

J2DVertex size is 16 bytes and since setVertexBytes has a limit of 4KB we can have maximum of 250 J2DVertex elements. I have created https://bugs.openjdk.java.net/browse/JDK-8261650 to add the comment.

Thanks,
Jay